docs/DATABASE_FORMAT.md

```diff
@@ -0,0 +1,114 @@
+# Instructions and Standards for the Development of Organized Logic Databases  
+  
+  
+## REGIONS, AREAS, AND NODES  
+There exist 3 tiers of "Location" in the Randovania logic database - Regions, Areas, and Nodes.  
+  
+Regions represent large swathes of the game's world, and a standard game will only have a few of these, often less than ten. Regions contain many areas grouped together. The criteria for what is and isn't its own region are mostly arbitrary. Generally, areas within the same region should be geographically adjacent, rather than being disjoint, and regions should have common themes. In addition, regions are usually drawn up as they are demarcated in the game being mapped. Regions usually have only a few connections to other regions, such as elevators, portals, or teleporters, though standard docks can still connect rooms between two different regions.  
+  
+Areas represent a single bite-sized chunk of the game, usually only a single room. The term "room" is often used interchangeably with area. Areas should remain true to the game - avoid excluding areas, even if you deem them irrelevant to randomization. Often, areas aren't given explicit or canonical names, and in this case it will be up to the database author to choose an appropriate name for a given area.  
+  
+Nodes represent a single point in a given area. Nodes are primarily used to represent pickups a player can collect, events a player can trigger, and docks to other areas. In addition, there exist some special nodes types, which are often game specific. The generic node, which has no specific game object associated with it, can be used for organizational purposes.  
+  
+  
+## THE RESOURCE DATABASE  
+In addition to mapping the geography of the game, it is necessary to define the different parameters a player is expected to progress within. The resource database defines tools a player has to progress, and the requirements they may be asked to overcome.  
+  
+Resources include items the player can obtain, tricks they can perform, events that can be triggered, and other such information.  
+  
+  
+## PICKUPS, DOCKS, AND EVENTS  
+Pickups represent a permanent inventory item which the player can acquire.  
+  
+Events represent a one-time, irreversible action the player can take which changes the game's logic. Events can only be performed once a game, and cannot be undone.  
+  
+Docks represent connections between two rooms. Docks should represent standard doors or connections, linking two geographically adjacent game areas. While it is possible for a dock to connect two rooms from different regions, special devices such as portals, elevators, fast-travel beacons, and other long-distance or inter-regional travel mechanisms should be represented as Teleporters.  
+  
+  
+## DEFINING LOGIC  
```

Comment:
Logic -> Requirements

Logic is the colloquial term, which means everything and nothing so we avoid it entirely.
(except there's a Logic class, which does almost nothing lol).

Of course you can use the term logic in the introduction.